_includes/resources.html

```diff
@@ -0,0 +1,34 @@
+---
+title: resources
+permalink: "/resources/"
+layout: default
+---
+
+<div class="resources">
```

Comment:
Looks like you have tabs for a lot of these files - can you convert to spaces and make indentations 2 spaces instead of 4 to match the rest of the files?